install/gcp/installer/util/utils.py

```diff
@@ -109,20 +109,50 @@ def _print_banner(border_symbol, edge_symbol, corner_symbol,
 
 
 def get_forseti_version():
-    """Get Forseti version from version file.
+    """Get the current Forseti version. If the current version is a branch
```

Comment:
The head of the doc string should be one line.

```
"""Get the current Forseti version. 

If the current version is a branch or a tag, we will return it. Otherwise we will read the __init__ file for the latest version.
"""
```

Reply:
Updated.